File: server/internal/entities/card.go
package entities

type Card struct {
	PlaceId   int      `json:"placeId"`
	Category  Category `json:"category"`
	Front     string   `json:"front"`
	InfoFront string   `json:"frontInfo"`
	Back      string   `json:"back"`
	InfoBack  string   `json:"backInfo"`
}

File: server/src/main/java/com/worldquiz/entities/Card.java
/* (C)2026 */
package com.worldquiz.entities;

import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_NULL)
public record Card(
        int placeId,
        Category category,
        String front,
        String infoFront,
        String back,
        String infoBack) {}
